src/2024h2/next-solver.md

```diff
@@ -0,0 +1,73 @@
+# Next-generation trait solver
+
+| Metadata | |
+| --- | --- |
+| Owner(s) | [@lcnr] |
+| Teams | [Types] |
+| Status | WIP |
```

Comment:
In progress?

Reply:
idk, it's what's used in the `impl-trait-everywhere` doc :thinking: :shrug:

Reply:
OH lol i think i misread this as "TODO fill in status" not "literally in progress"